### PR TITLE
BAU: refactor organisation logo CSS

### PR DESCRIPTION
**Pull request by robinwhittleton**
Desktop mode uses a function to generate the background image, mobile code is hardcoded. This moves the mobile code into the same function.

### CODE REVIEW

**Comment by dsingleton** [comment]
SASS change looks good, have you tested on a page using those styles? (it looks like it's only [campaign pages in frontend](https://github.com/search?utf8=%E2%9C%93&type=Code&q=organisation-logo-+repo:alphagov/business-support-finder+repo:alphagov/calculators+repo:alphagov/calendars+repo:alphagov/collections+repo:alphagov/contacts-admin+repo:alphagov/contacts-frontend+repo:alphagov/courts-frontend+repo:alphagov/design-principles+repo:alphagov/email-alert-frontend+repo:alphagov/feedback+repo:alphagov/finder-frontend+repo:alphagov/frontend+repo:alphagov/government-frontend+repo:alphagov/homepage-frontend+repo:alphagov/info-frontend+repo:alphagov/licence-finder+repo:alphagov/manuals-frontend+repo:alphagov/smart-answers+repo:alphagov/specialist-frontend+repo:alphagov/whitehall)

**Comment by alextea** [comment]
Could these changes also be applied to [whitehall](https://github.com/alphagov/whitehall/blob/master/app/assets/stylesheets/frontend/helpers/_organisations.scss)?

**Comment by fofr** [comment]
It's hard to judge/test the affect this has. I would :heart: a style guide page that demonstrates these rules, this would allow wraith regression tests too.

**Comment by dsingleton** [comment]
A styleguide would be useful. It would need to cover all the things on static not currently in components, which is pretty much [things from the wrapper layout](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/static.scss), that we're trying to get rid of - by converting to components or moving directly into apps.

There's a question about how much effort a style guide would be vs the 'getting rid' work though.

**Comment by fofr** [comment]
For the migration of HTML publications we're looking at porting organisation logos from Whitehall into government-frontend, this will probably involve creating a component.

With that in mind, I'm happy for this change to be merged.

File: app/assets/stylesheets/helpers/_organisations.scss [hunk 1]
```diff
@@ -156,53 +156,33 @@
 /* Crests */
 
 @mixin organisation-logo-image($crest) {
-  background: image-url($crest) no-repeat 8px 0;
+  background: image-url('crests/#{$crest}_18px.png') no-repeat 8px 0;
+  @include media-down(mobile) {
```
Comment (gemmaleigh) [inline app/assets/stylesheets/helpers/_organisations.scss]: Is it possible to swap these two, so the 13px crest is defined first and `@include media(tablet)` is used to increase the crest size for larger screens?

I'd like to remove `media-down` where we can.

**Comment by robinwhittleton** [comment]
@gemmaleigh happy with this change?

**Comment by gemmaleigh** [comment]
:+1: thanks @robinwhittleton.

**Pull request merged**